site/resources/js/site.js

```diff
@@ -188,6 +188,10 @@ window.addEventListener('DOMContentLoaded', function() {
     else {
       switcher.scale = 'medium';
     }
+    let scaleDropdown = document.querySelector('#switcher-scale');
```

Comment:
since this is part of the event listener matching a media query for max-width 768px won't this be fired up every time the window sizes in that viewport range? Does this mean the query selector will fire up multiple times?
Maybe this should be moved outside the window.matchMedia listener.  

```
 var scaleMQL = window.matchMedia('(max-width: 768px)');
  function handleScaleMQLChange() {
    if (scaleMQL.matches) {
      switcher.scale = 'large';
    }
    else {
      switcher.scale = 'medium';
    }
    let scaleDropdown = document.querySelector('#switcher-scale');
    if (scaleDropdown) {
      setDropdownValue(scaleDropdown, switcher.scale);
    }
  }
  scaleMQL.addListener(handleScaleMQLChange);
```

Reply:
Oh it totally could be moved out. I'll do that.

Reply:
Wait I lied, it has to run inside of the listener; some pages have it, some don't, and we get a new dropdown every time we move to a new page (asynchronously).

Reply:
I'll cache it in FastLoad's listener!